src/components/AddToMap.js

```diff
@@ -20,19 +16,29 @@ const BUSINESS_UPDATE_MAP_INFO = gql`
   }
 `
 
-function AddToMap() {
-  const [businessUpdateMapInfo] = useMutation(BUSINESS_UPDATE_MAP_INFO)
+function useStateWithDep(defaultValue) {
```

Comment:
I'd like to understand this a bit more. Why are we using a custom hook here for the states for `titleVal`, `latitudeVal` etc?